src/server.js

```diff
@@ -0,0 +1,7 @@
+import app from './app';
+
+const server = app.listen(3000, () =>
+  console.log('App listening on port 3000!....')
+);
```

Comment:
Create the .env file and add the PORT variable.

Reply:
Someone else has this task of creating the .env file, So I am not supposed to do that.